### PR TITLE
Fixed #2103 - Gifs not loading in Emails module

### LINKED ISSUE #2103: Gifs not loading in Emails module
#### Issue

Gifs not loading in Emails module
#### Expected Behavior

They should load.
#### Actual Behavior

They didn't.
#### Steps to Reproduce

1.Go to Emails module
#### Context

low
#### Your Environment
- SuiteCRM Version used: 7.7.4
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): chrome 52.0.2743.116 m
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04): win10
  zjd

### PR DESCRIPTION
**Pull request by shogunpol**
Attachement icon is load in Emails module
## Description

Attachement icon is load in Emails module
References issue #2103 

Make connection for work icon display.

1.Go to Emails module
2.See attachement icon

## Motivation and Context

Improve apearience of email module
## How To Test This

Repeat steps from replication bug section
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/SugarTheme/SugarTheme.php [hunk 1]
```diff
@@ -900,12 +900,12 @@ public function getImageURL(
         $imageName,
         $addJSPath = true
         ){
-        if ( isset($this->_imageCache[$imageName]) ) {
+      /*  if ( isset($this->_imageCache[$imageName]) ) {
```
Comment (lewisje1991) [inline include/SugarTheme/SugarTheme.php]: Why is this commented out?

File: modules/Emails/javascript/Email.js [hunk 2]
```diff
@@ -291,7 +291,7 @@ function multiFiles( list_target){
         imgElement.setAttribute("width","16");
 
         var new_row_button_embed = document.createElement("img");
-        new_row_button_embed.setAttribute("src", "index.php?entryPoint=getImage&themeName="+SUGAR.themes.theme_name+"&imageName=attachment.gif");
+        new_row_button_embed.setAttribute("src", "index.php?entryPoint=getImage&themeName="+SUGAR.themes.theme_name+"&imageName=attachment.svg");
```
Comment (lewisje1991) [inline modules/Emails/javascript/Email.js]: I believe changing these to .svg will cause the same issue in other themes?

Comment (shogunpol) [inline modules/Emails/javascript/Email.js]: @lewisje1991 , This functionality is work for all themes. There is backward compatibility previous themes

File: themes/SuiteP/css/yui.css [hunk 3]
```diff
@@ -6133,3 +6133,12 @@ a.sugar-tab-close, .yui-navset .yui-nav a.sugar-tab-close, .yui-navset .yui-nav
 .yui-layout .yui-layout-clip {
     padding:10px 000;
 }
+
+.yui-dt-label .image {
+    width: 16px;
+    height: auto;
+}
+
+#yui-dt15-th-hasAttach-liner {
```
Comment (shogunpol) [inline themes/SuiteP/css/yui.css]: @lewisje1991 , it is changed

File: include/SugarTheme/getImage.php [hunk 4]
```diff
@@ -38,6 +38,9 @@
  * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  ********************************************************************************/
 
+if(stristr($_REQUEST['imageName'], 'attachment') !== FALSE) {
```
Comment (gymad) [inline include/SugarTheme/getImage.php]: Please use lover case `false` as it should match the PSR standards:
see https://www.php-fig.org/psr/psr-2/#25-keywords-and-truefalsenull

File: include/SugarTheme/SugarTheme.php [hunk 5]
```diff
@@ -897,7 +897,7 @@ public function getImageURL(
         $addJSPath = true
         ){
         if ( isset($this->_imageCache[$imageName]) ) {
-            if ( $addJSPath )
+            if ($addJSPath)
```
Comment (gymad) [inline include/SugarTheme/SugarTheme.php]: Not up to standards, missing `{`, `}` brackets.

File: include/SugarTheme/getImage.php [hunk 6]
```diff
@@ -110,6 +113,8 @@
 // now send the content
 if ( substr($filename,-3) == 'gif' )
     header("Content-Type: image/gif");
+elseif ( substr($filename,-3) == 'svg' )
```
Comment (gymad) [inline include/SugarTheme/getImage.php]: Not up to standards, missing `{`, `}` brackets.

File: include/SugarTheme/SugarTheme.php [hunk 7]
```diff
@@ -897,10 +897,12 @@ public function getImageURL(
         $addJSPath = true
         ){
         if ( isset($this->_imageCache[$imageName]) ) {
-            if ( $addJSPath )
+            if ($addJSPath) {
                 return getJSPath($this->_imageCache[$imageName]);
-            else
+	    }
```
Comment (gymad) [inline include/SugarTheme/SugarTheme.php]: wrong indents

**Comment by samus-aran** [comment]
This has been resolved.

**Pull request closed without merging**